.tool-versions

```diff
@@ -1,2 +1,2 @@
 ruby 3.1.6
```

Comment:
Why do we need to keep the Ruby version on 3.1.6?

Reply:
This is a publicly available gem. We wanted to keep the supported set of ruby versions as big as possible. And 3.1 is the lowest supported one (I think).